security_fixes.sh

```diff
@@ -38,10 +39,10 @@ then
 	echo "on a MiSTer system."
 	exit 1
 fi
-if [ "$SSH_CLIENT" == "" ]
+if [ "$(tty)" == "not a tty" ]
```

Comment:
since the goal is to check that the user will be able to see the messages and to input data, wouldn't it be better to check if stdin (0), stdout (1) and stderr (2) are connected or not to a terminal?
```sh
if [[ ! (-t 0 && -t 1 && -t 2) ]] ; then
```

Reply:
Either would probably be sufficient. But your suggestion looks more idiomatic.

Reply:
Thanks @makigumo